src/Support/LaravelFinder.php

```diff
@@ -47,7 +49,12 @@ public function environmentFilePaths(): array
     {
         foreach($this->environmentFiles as $envFile) {
             if(!file_exists( $envFile )) {
-                File::copy(base_path() . "/vendor/worksome/envy/config/.env.example", $envFile);
+                $fileName = substr($envFile, strrpos($envFile, '/') + 1);
+                if ($fileName == '.env.example') {
+                    File::copy(base_path() . "/vendor/worksome/envy/config/$fileName.".substr(app()->version(),0,1), $envFile);
+                } else {
+                    File::copy(base_path() . "/vendor/worksome/envy/config/$fileName", $envFile);
```

Comment:
Will raise the "failed to open stream" warning if the file does not exist. Unpredictable behavior.

Reply:
You're right. But the goal here was to resolve the issue for the main env file (.env.example), which this PR will do that. 
What's your suggestion to the issue? How do you think we can get rid of this issue entirely for all the files?